<ui version="4.0" >
 <comment> * OpenRPT report writer and rendering engine
 * Copyright (C) 2001-2012 by OpenMFG, LLC
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 * Please contact info@openmfg.com with any questions on this license.</comment>
 <class>ReportParameterList</class>
 <widget class="QDialog" name="ReportParameterList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameter List</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="_lblParameters" >
       <property name="text" >
        <string>Parameters:</string>
       </property>
       <property name="buddy" >
        <cstring>_lbParameters</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="_lbParameters" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="_btnClose" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="_line" >
       <property name="frameShape" >
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnAdd" >
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnEdit" >
       <property name="text" >
        <string>&amp;Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>86</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>_lbParameters</tabstop>
  <tabstop>_btnAdd</tabstop>
  <tabstop>_btnEdit</tabstop>
  <tabstop>_btnDelete</tabstop>
  <tabstop>_btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ReportParameterList</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>321</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
